### DISCUSSION TITLE
I've created https://github.com/OpenVSLAM-Community/DBoW2.

### DISCUSSION THREAD

**Discussion by tzjZhengJie** [stella-cv/stella_vslam#423] [Q&A] [answered]
I've created https://github.com/OpenVSLAM-Community/DBoW2.

_Originally posted by @ymd-stella in https://github.com/stella-cv/stella_vslam/issues/17#issuecomment-770395328_
      
Hi, your current DBoW2 Link is currently not working. Are there any new replacement?

**Comment by ymd-stella** [comment, marked as answer]
Use https://github.com/stella-cv/FBoW.
